tests/lib/snaps/test-snapd-pulseaudio/files/bin/pawrap

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Well, not without rearranging the tests under a single snap. I wanted different snaps since the pulseaudio test will always be just pulseaudio, but audio-playback is expected to be extended to include pipewire, etc. Merging everything into a shared 'test-snapd-audio' snap is possible, but messier for this reason IMO. Furthermore, since these require compiling code, they are built out of bzr branches in LP so I can't do symlinks/hardlinks/similar since they are distinct trees in LP and the code here is merely a mirror of that code (I'm not entirely sure why this is the current methodology, but I'm told this is the way to do it)).

Reply:
this is fine for now, we discussed having a test-snapd repo for all of these snaps that makes sure the snaps are updated and such and have that repo be independent from snapd so I guess we can leave this here for now until we do that switch.